src/helpers/analyticsEvents/highlighting/CreateNote.ts

```diff
@@ -0,0 +1,21 @@
+import * as selectContent from '../../../app/content/selectors';
+import { AnalyticsEvent } from '../event';
+
+const eventName = 'REX highlighting - create note';
+
+export const track = (
+  book: ReturnType<typeof selectContent['bookAndPage']>['book'],
+  color: string
+): AnalyticsEvent | void => {
+  const slug = book ? book.slug : 'unknown';
```

Comment:
lets do the url path instead of just the book slug, check some other analytics events for examples